src/main/java/com/sigopt/model/Experiment.java

```diff
@@ -118,6 +118,26 @@ public Integer getFolds() {
         return Experiment.delete().addPathComponent("id", id);
     }
 
+    /**
+    *   LinkedResource is used for a resource linked strongly to another e.g.
```

Comment:
I like your thinking! I would think something like PropertyResource, since stopping criteria is a property while observations are a subresource. This is also because `linked` has such a strong connotation for programmers already and is tied to object structure, lists, etc.